plugins/nginx-vhosts/install

```diff
@@ -22,45 +22,12 @@ ssl_session_cache shared:SSL:20m;
 ssl_session_timeout 10m;
 
 ssl_ciphers EECDH+AES128:RSA+AES128:EECDH+AES256:RSA+AES256:EECDH+3DES:RSA+3DES:!MD5;
-ssl_protocols TLSv1.2 TLSv1.1 TLSv1;
-ssl_prefer_server_ciphers on;
```

Comment:
I have removed the `ssl_protocols` and `ssl_prefer_server_ciphers` directives because they are already included in the default `nginx.conf` of nginx 1.8.0 (the current stable version). Otherwise, nginx would throw an `[emerg] directive is duplicate` error.